### PR TITLE
#180705046 fixes

### PR DESCRIPTION
**Pull request by SalShai**
## Feature description
- Fix: Notification to Authorized Signatory.
- Modify Work Contract Doc to remove from Customization
- Correct Work Contract Print Format.

## Solution description
- Use Notification List for "Notification to Authorized Signatory"
- Removed Border in print Format Signature Section.

## Output screenshots (optional)

Screen Shot 2021-12-30 at 9 54 41 AM

## Areas affected and ensured
- Workflow consists of Accept and Reject actions.

## Is there any existing behavior change of other features due to this code change?
No.

## Was this feature tested on all the browsers?
  - [x] Chrome
  - [x] Safari

### CODE REVIEW

File: one_fm/hiring/doctype/work_contract/work_contract.py [hunk 1]
```diff
@@ -41,12 +41,14 @@ def update_on_workflow_state(self):
 		if self.workflow_state == 'Send to Authorised Signatory':
 			self.validate_attachments()
 			self.validate_authorized_signatory()
-			email_authority_for_signature(self)
+			#email_authority_for_signature(self)
```
Comment (pjamsheer) [inline one_fm/hiring/doctype/work_contract/work_contract.py:44]: If this line is not needed, you can remove it instead of commenting it

Comment (pjamsheer) [reply one_fm/hiring/doctype/work_contract/work_contract.py:44]: @SalShai What is one_fm/hiring/notification/__init__.py

Comment (SalShai) [reply one_fm/hiring/doctype/work_contract/work_contract.py:44]: I created a Notification List for Work Contract to sent Notifiction for Approval.

Comment (SalShai) [reply one_fm/hiring/doctype/work_contract/work_contract.py:44]: I used this method because Workflow didn't help.

Comment (pjamsheer) [reply one_fm/hiring/doctype/work_contract/work_contract.py:44]: Okay

**Comment by pjamsheer** [comment]
@SalShai it is on top of master-13 now

**Review by pjamsheer — APPROVED** [review]

**Pull request merged**